src/templates/steps-links-edit.html

```diff
@@ -128,7 +128,7 @@
                     <span class='input-group-text'>{{ 'Add a link'|trans }}</span>
                 </div>
                 <input aria-label='{{ 'Add a link'|trans }}' type='text' data-autocomplete='links' id='linkinput' class='form-control' data-id='{{ Entity_id }}' placeholder='{{ 'from the database'|trans }}' />
-                <div class='input-group-append'>
+                <div class='input-group-append' style='max-width:33%;'>
```

Comment:
Not sure why `mw-25` bootstrap class wouldn't work in my hands here :/ [doc](https://getbootstrap.com/docs/4.0/utilities/sizing/)